test/unit/specs/hive/basic-store.js

```diff
@@ -0,0 +1,2237 @@
+import { expect } from 'chai'
+import { describe, it } from 'mocha'
+
+import Backendless, { APP_PATH, forTest, prepareMockRequest } from '../../helpers/sandbox'
+import { HiveTypes } from '../../../../src/hive/constants'
+import { HiveStore } from '../../../../src/hive/stores/base-store'
+
+describe('Hive Store Class', function() {
+  forTest(this)
+
+  const hiveName = 'test'
+  const storeKey = 'testStoreKey'
+
+  const getStoreUrl = storeName => `${APP_PATH}/hive/${hiveName}/${storeName}`
+
+  describe('Store Creating', () => {
+    it('without key', async () => {
+      const keyValueStore = Backendless.Hive(hiveName).KeyValueStore()
+      const listStore = Backendless.Hive(hiveName).ListStore()
+      const mapStore = Backendless.Hive(hiveName).MapStore()
+      const setStore = Backendless.Hive(hiveName).SetStore()
+      const sortedSetStore = Backendless.Hive(hiveName).SortedSetStore()
+
+      expect(keyValueStore).to.be.instanceof(HiveStore)
```

Comment:
need to clean up all redundant checks, if make a request for a specific type and we can see a correct request it should be enough 

does it make sense?

Reply:
done